core/selection.js

```diff
@@ -531,6 +531,9 @@
 					return;
 				}
 
+				// Cache selected widgets.
+				var selectedWidgets = editor.widgets.selected.slice();
```

Comment:
`editor.widgets` is not always present! Due to not checking at least _several hundreds_ of tests are failing.